src/test/scala/za/co/absa/faDB/examples/ExampleSuite.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.faDB.examples
+
+import org.scalatest.funsuite.AnyFunSuite
+
+class ExampleSuite extends AnyFunSuite {
```

Comment:
Why is this commented out?

Reply:
The whole examples folder will go It's just a remnant of the POC development process.
(I committed it with the first batch just not to loose the code before vacation. Will do a cleaning before release.)